config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/environment/EnvironmentBundleBuilder.java

```diff
@@ -9,37 +9,79 @@
 import com.ca.apim.gateway.cagatewayconfig.beans.Bundle;
 import com.ca.apim.gateway.cagatewayconfig.config.loader.EntityLoader;
 import com.ca.apim.gateway.cagatewayconfig.config.loader.EntityLoaderRegistry;
+import com.ca.apim.gateway.cagatewayconfig.util.environment.EnvironmentConfigurationUtils;
 
 import javax.inject.Inject;
 import javax.inject.Singleton;
+import java.io.File;
+import java.util.Collection;
 import java.util.Map;
 
+import static com.ca.apim.gateway.cagatewayconfig.environment.EnvironmentBundleCreationMode.APPLICATION;
+import static com.ca.apim.gateway.cagatewayconfig.util.properties.PropertyConstants.PREFIX_ENV;
+
 @Singleton
+@SuppressWarnings("squid:S2083") // This warn relates to path injection attacks - however, paths here are never changed by end users and all self contained into docker containers.
```

Comment:
what are different ways to provide env variables / files, can you please mention them? I believe environment.properties is one.

Reply:
This is not related only to environment variables to cluster properties, this is related to environment at all (jdbc, jms, etc) and it's mentioning in the javadoc that collect values from environment vars or files.
I think file names and definitions are more for wiki sections, otherwise we will only add one more place to get easily outdated.

Reply:
yeah that makes sense..